embulk-core/src/main/java/org/embulk/EmbulkRunner.java

```diff
@@ -373,11 +374,8 @@ private String runLiquid(
             final Map<String, Object> templateParams,
             final String templateIncludePath) {
         // TODO: Check if it is required to process JRuby options.
-        // Not |ScriptingContainerDelegate.LocalContextScope.SINGLETON| to narrow down considerations.
-        final ScriptingContainerDelegate localJRubyContainer = ScriptingContainerDelegateImpl.create(
-                EmbulkRunner.class.getClassLoader(),
-                ScriptingContainerDelegate.LocalContextScope.SINGLETHREAD,
-                ScriptingContainerDelegate.LocalVariableBehavior.PERSISTENT);
+        final ScriptingContainerDelegate localJRubyContainer =
+                LazyScriptingContainerDelegate.withGems(rootLogger, this.embulkSystemProperties);
 
         localJRubyContainer.runScriptlet("require 'liquid'");
```

Comment:
Looks like `localJRubyContainer` can be `null`.

Reply:
Thanks! Added a (very simple) error messaging there also with `gem` and `bundle` in ef7af455559cec2eaf52ed5541bc503de7ae3cf0. We may want to improve those error handling later...